src/components/OrderPanel/BookingDatesForm/BookingDatesForm.js

```diff
@@ -598,8 +704,57 @@ export const BookingDatesFormComponent = props => {
               onClose={() => {
                 setCurrentMonth(startDate || endDate || startOfToday);
               }}
+              onChange={values => {
+                const { startDate, endDate } = values || {};
+                const parsedStart = startDate
+                  ? getStartOf(timeOfDayFromLocalToTimeZone(startDate, timeZone), 'day', timeZone)
+                  : startDate;
+                const parsedEnd = endDate
+                  ? getStartOf(timeOfDayFromLocalToTimeZone(endDate, timeZone), 'day', timeZone)
+                  : endDate;
+                const endDateForAPI =
+                  parsedEnd && isDaily ? getExclusiveEndDate(parsedEnd, timeZone) : parsedEnd;
+                if (availabilityType && availabilityType == 'multipleSeats') {
+                  formApi.change('seats', 1);
+                }
+                onHandleFetchLineItems({
+                  values: {
+                    startDate: parsedStart,
+                    endDate: endDateForAPI,
+                    seats: availabilityType && availabilityType === 'multipleSeats' ? 1 : undefined,
+                  },
+                });
+              }}
             />
 
+            {availabilityType == 'multipleSeats' ? (
```

Comment:
`availabilityType === AVAILABILITY_MULTIPLE_SEATS`

I think this should be extracted to a new variable: 
`const useMultipleSeats = availabilityType === AVAILABILITY_MULTIPLE_SEATS;`